src/fsharp/FSharp.Core/async.fsi

```diff
@@ -93,14 +120,30 @@ namespace Microsoft.FSharp.Control
         /// in the corresponding state once the computation terminates (produces the result, throws exception or gets canceled)</returns>
         ///
         /// <category index="0">Starting Async Computations</category>
-        /// 
-        /// <example-tbd></example-tbd>
+        ///
+        /// <example id="start-as-task-1">
+        /// <code lang="fsharp">
+        /// printfn "A"
+        ///
+        /// let t =
+        ///     async {
+        ///         printfn "B"
+        ///         do! Async.Sleep(1000)
+        ///         printfn "C"
+        ///     } |> Async.StartAsTask
+        ///
+        /// printfn "D"
+        /// t.Wait()
+        /// printfn "E"
+        /// </code>
+        /// Prints "A", "D" immediately, then "B" quickly, then "C", "E" in 1 second.
```

Comment:
There's really a race between D and B, with it much more likely that `D` will print first though in theory they might print in either order.  Best to say that